engine/src/main/java/org/terasology/registry/InjectionHelper.java

```diff
@@ -31,7 +32,28 @@
 public final class InjectionHelper {
     private static final Logger logger = LoggerFactory.getLogger(InjectionHelper.class);
 
-    private InjectionHelper() {
+    public InjectionHelper(Context context) {
```

Comment:
oops, that was a leftover from a refactor to make it a non static helper class which I reverted later on.